platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/problems/ConfigurationCacheProblems.kt

```diff
@@ -365,6 +382,7 @@ class ConfigurationCacheProblems(
             when {
                 seenSerializationErrorOnStore && deferredProblemCount == 0 -> log("Configuration cache entry discarded due to serialization error.")
                 seenSerializationErrorOnStore -> log("Configuration cache entry discarded with {}.", problemCountString)
+                cacheAction == Store && shouldDegradeGracefully() -> log("Configuration caching disabled${degradationSummary()}")
```

Comment:
💅  Saying "Configuration _caching_" is not consistent with the rest of the messages